docs/sources/guides/whats-new-in-v6-7.md

```diff
@@ -74,13 +80,17 @@ rendering_timeout = 10s
 concurrent_render_limit = 10
 ```
 
-### Export dashboard as pdf
-This feature allows you to export the dashboard as a PDF document. All dashboard panels will be rendered as images and added into the PDF document.
+### Export dashboard as PDF
+
+This feature allows you to export a dashboard as a PDF document. All dashboard panels will be rendered as images and added into the PDF document.
 
 ### Report landscape mode
+
 You can now use either portrait or landscape mode in your reports. Portrait will render three panels per page and landscape two.
 {{< docs-imagebox img="/img/docs/enterprise/reports_create_new.png" max-width="1024px" caption="New report" >}}
 
+[Reporting]({{< relref "../enterprise/reporting/" >}}) has been updated as a result of this change.
+
 ### Azure OAuth Team Sync support
-When setting up OAuth with Microsoft Azure AD it’s now possible to sync Azure groups with Teams in Grafana.
-[Learn more]({{< relref "../auth/azuread/#team-sync-enterprise-only" >}}).
+When setting up OAuth with Microsoft Azure AD, you can now sync Azure groups with Teams in Grafana.
+Learn more in [Azure AD OAuth2 authentication]({{< relref "../auth/azuread/#team-sync-enterprise-only" >}}).
```

Comment:
Is this the right place to link to? Seems like the Team sync topic might make more sense.